src/imageLoader/createImage.js

```diff
@@ -186,6 +186,13 @@ function createImage (imageId, pixelData, transferSyntax, options) {
         image.windowWidth = maxVoi - minVoi;
         image.windowCenter = (maxVoi + minVoi) / 2;
       }
+
+      // overlays
+      const overlays = cornerstone.metaData.get('overlayPlaneModule', imageId);
+      if (overlays && overlays.length > 0) {
+        image.overlays = overlays;
+      }
+
```

Comment:
If we leave this off of `createImage` for now (we can discuss later), then I feel comfortable merging this after a quick gut check from @swederik. This should give us just enough to test an `OverlayTool` -- we can iterate and clean this and the corresponding tool up after we get an example and working proof-of-concept.

Reply:
if you leave it off createImage how is it going to be available for the tool ?

Reply:
Tool's can grab metadata by asking registered metadata providers:

`cornerstone.metaData.get('overlayPlaneModule', imageId);`

You can see an example of us doing it in the [current length tool src](https://github.com/cornerstonejs/cornerstoneTools/blob/master/src/tools/annotation/LengthTool.js#L126-L129).

Reply:
got it ..

Reply:
removed createimage.js from the PR

Reply:
yeah that's what I understood
so I removed createimage.js changes from the PR - they are not needed
and added changes to your tool to read directly from metadata